test/svm/SvmSpoke.InstructionParams.ts

```diff
@@ -0,0 +1,132 @@
+import * as anchor from "@coral-xyz/anchor";
+import * as crypto from "crypto";
+import { Keypair, PublicKey } from "@solana/web3.js";
+import { common } from "./SvmSpoke.common";
+
+const { provider, program, connection, assertSE, assert } = common;
+
+describe("svm_spoke.instruction_params", () => {
+  anchor.setProvider(provider);
+
+  const payer = (anchor.AnchorProvider.env().wallet as anchor.Wallet).payer;
+
+  // We use different caller in each test as instructionData seed is derived from initializer's address.
+  let caller: Keypair;
+  let instructionParams: PublicKey;
+
+  const initializeInstructionParams = async (totalSize: number) => {
+    const ix = await program.methods
+      .initializeInstructionParams(totalSize)
+      .accounts({ signer: caller.publicKey, instructionParams })
+      .instruction();
+    await anchor.web3.sendAndConfirmTransaction(connection, new anchor.web3.Transaction().add(ix), [caller]);
+  };
+
+  const writeInstructionParams = async (data: Buffer) => {
+    // Should not exceed message size limit when writing to the data account
+    const maxInstructionParamsFragment = 900;
```

Comment:
This limit needs to be taken into consideration when calculating the rootBundles off-chain otherwise we might have non-claimable bundles in solana right?